robotidy/transformers/NormalizeSeparators.py

```diff
@@ -1,9 +1,10 @@
 from robot.api.parsing import KeywordCall, Token
 
 try:
-    from robot.api.parsing import InlineIfHeader
+    from robot.api.parsing import InlineIfHeader, ReturnStatement
```

Comment:
ReturnStatement is ``RETURN`` from 5.0. Robotidy supports RF >=4.0 so we're using try import error to work with both 4.0 and 5.0.